docs/flashbots-mev-share/release-notes/2023-09.mdx

```diff
@@ -0,0 +1,18 @@
+# September 2023
+
+## default logs hint
+
+A new hint which shares a subset of logs, as described in the [stable configuration](/flashbots-protect/mev-share#stable-configuration).
+
+Previously these logs were shared by default. Now, users of the private transaction API, Protect RPC, and bundle relay can also request to share these logs when customizing their settings.
+
+## simple-limit-order-bot
+
+New tutorial for searching on MEV-Share, using a flashloan arbitrage bot as an example.
+
+- [Step by step tutorial with full code examples](/flashbots-mev-share/searchers/tutorials/flash-loan-arbitrage/introduction)
+- [Open source bot repository](https://github.com/flashbots/simple-blind-arbitrage)
+
+## SSE stability updates
+
+Several operational and architectural improvements which make the stream more reliable and up-to-date.
```

Comment:
We have improved the stability of our SSE stream endpoint for increased stability and observed a significant decrease in client timeouts and connecting hanging as a result

We enabled transparent proxying of requests to SSE stream and updated SSE stream service to better support and scale under dynamic workloads and handle a greater number of client connections

Feel free to use any or none of the above as you wish